src/orders/dto/findAllOrder.dto.ts

```diff
@@ -6,3 +6,12 @@ export class FindAllOrderDto extends PaginationQueryDto {
   @IsOptional()
   user_id: string;
 }
+
+export class GetOrdersFilterDto extends PaginationQueryDto {
```

Comment:
if you use pagination DTO in controller and service, there is no need to extends it anymore

Reply:
fixed

Reply:
fixed